src/main/jbake/content/jsf-intro009.adoc

```diff
@@ -16,10 +16,7 @@ For more information on Jakarta Server Faces technology, see
 
 * Jakarta Server Faces 2.3 specification:
 +
-`http://jcp.org/en/jsr/detail?id=372`
-* Jakarta Server Faces project website:
+`https://jakarta.ee/specifications/faces/2.3/`
+* Mojarra website:
 +
-`https://javaserverfaces.github.io/users.html`
-
-For additional samples, see the GlassFish samples at
-https://github.com/javaee/glassfish-samples/tree/master/ws/javaee8.
```

Comment:
We're in the process of contributing the GlassFish samples repository to Eclipse.
Let's leave this reference in for now.

Reply:
Oh I thought I read they weren't going to be contributed as they were outdated. I readded the link.

Reply:
It's the Java EE samples that are outdated and won't be contributed.
These GlassFish samples may also be outdated but we're still contributing them!  :-)